models/calendar.yml

```diff
@@ -186,7 +186,16 @@
         title: Intermission
         emoji: 🕣
 
-    - 8:30 PM: Performer TBA
+    - 8:30 PM:
+        type: Performance
+        emoji: ⚡
+        title: Sounds All Around
+        description: I like to make interactive and immersive art.
```

Comment:
You can run this locally by the way! Just type `npm install` then `npm start` and you should be good to go.